config/i18n.json

```diff
@@ -317,7 +317,7 @@
     "RarityTier": "Shows items based on their rarity tier.",
     "Reforgeable": "Shows items that can be reforged at the Gunsmith.",
     "Release": "Available from release <i>{{release}}</i>",
-    "Shiny": "Shows BRAVE weapons with a limited-edition ornament.",
+    "Holofoil": "Shows weapons with a holofoil ornament.",
```

Comment:
It's not an ornament anymore, is it?

Reply:
so older ones are still ornaments, but new ones are not...